### ISSUE TITLE
infer table output format from path suffix in headless

### ISSUE THREAD

**Issue by aafkegros** [ilastik/ilastik#2864]
if i run a headless with a csv table filename (ex: `--table_filename {dataset_dir}/ilastik_output/{nickname}_fullproj.csv`) but have not set the output format of the ilp to csv in the GUI, ilastik will break and tell me to change the output format. 
Can we instead infer output format from the table path suffix in headless?

**Comment by k-dominik** [comment]
that and make csv the default maybe (as this is what most people want)...

**Comment by oanegros** [comment]
yes, hdf5 is nice if you need it, but default behaviour should be as generic as possible
